### PR TITLE
Update Images for Session 3

### PR DESCRIPTION
**Pull request by moritzkunzmann**
Replaces Lena image with image of brain CT and image of binary cell masks with microscopy image of neurons.

### CODE REVIEW

File: assignments/session1.tex [hunk 1]
```diff
@@ -55,7 +55,7 @@ \section{Intensity thresholding and Dice coefficient}
             \item Use ``\texttt{plt.imread('data/NIH3T3/gt/0.png')}'' to load the ground truth.
         \end{enumerate}
     \end{enumerate}
-    \item Commit and push your changes. Remember to this regularly!
+    \item Commit and push your changes. Remember to do this regularly!
```
Comment (kostrykin) [inline assignments/session1.tex:58]: Thanks for spotting this!

File: assignments/session3.tex [hunk 2]
```diff
@@ -18,7 +18,7 @@
 \begin{enumerate}
 \item Explore the Fourier transform by following these steps:
 \begin{enumerate}
-    \item Load the first image as given in the notebook, resize it to (256, 256) pixels using the \texttt{resize()} function, and show the result.
+    \item Load the first image\footnote{\url{https://www.cellpose.org/dataset}} as given in the notebook, resize it to (256, 256) pixels using the \texttt{resize()} function, and show the result.
```
Comment (kostrykin) [inline assignments/session3.tex:21]: I'm a bit concerned that some students might misinterpret this as "I have to download it myself".
```suggestion
    \item Load the first image\footnote{Source: \url{https://www.cellpose.org/dataset}} as given in the notebook, resize it to (256, 256) pixels using the \texttt{resize()} function, and show the result.
```
Would changing it like this maybe help, what you think? Or something different?

Same also for the other footnote below.

File: assignments/session3.tex [hunk 3]
```diff
@@ -44,10 +44,10 @@
 \begin{samepage}
 \item Study the role of the amplitude and the phase:
 \begin{enumerate}
-    \item Load the image \texttt{data/lena.png}.
+    \item Load the image \texttt{data/brain\_ct.png}\footnote{\url{https://brain-development.org/ixi-dataset}}.
```
Comment (kostrykin) [inline assignments/session3.tex:47]: See above.

What you think?
```suggestion
    \item Load the image \texttt{data/brain\_ct.png}\footnote{Source: \url{https://brain-development.org/ixi-dataset}}.
```

**Review by kostrykin — COMMENTED** [review]
Much thanks @moritzkunzmann!

A few comments inside.

PDF: https://github.com/BMCV/mobi-fs5-python-assignments/actions/runs/11107739499/artifacts/1995817080

xref https://github.com/BMCV/mobi-fs5-python/pull/4

File: assignments/session3.tex (hunk 2 above)
Comment (moritzkunzmann) [reply assignments/session3.tex]: Thanks for the suggestion, sounds better.

**Review by kostrykin — APPROVED** [review]

**Pull request merged**